CHANGELOG.rst

```diff
@@ -18,7 +17,6 @@ not released
    `default_event_duration` and `default_dayevent_duration` for an day-long 
    event
 * NEW Add `{uid}` property to template options in `--format`
-* FIX No warning whan importing event with Windows timezone format
```

Comment:
Ah. Found it. Did you decide it wasn't necessary, or was it lost resolving the conflict with master?